src/ResourceManagement/Sql/ElasticPoolActivityImpl.cs

```diff
@@ -3,137 +3,169 @@
+        ///GENMHASH:03526126BC38B1E6BD9561737558EC5D:1175667EA7EC5F56E7E123B5FF22F500
+        public string OperationId()
         {
-            this.resourceId = ResourceId.FromString(Inner.Id);
+            return this.Inner.OperationId.ToString();
         }
 
-        ///GENMHASH:579156BC87293AD83F70BBF347CFEF47:8ACA8E03B61F5FC45CB2DFEE3F51696A
-        public long RequestedStorageLimitInGB()
+        ///GENMHASH:8550B4F26F41D82222F735D9324AEB6D:42AE1A0453935D9BF88147F2F9C3EC20
+        public DateTime? StartTime()
         {
-            return Inner.RequestedStorageLimitInGB.GetValueOrDefault();
+            return this.Inner.StartTime;
         }
 
-        ///GENMHASH:3E38805ED0E7BA3CAEE31311D032A21C:A08302619254C2A4BBCEC7165449AD96
-        public string Name()
+        ///GENMHASH:ACA2D5620579D8158A29586CA1FF4BC6:003E1843567E7760DFADC015F30E6AF4
+        public string Id()
         {
-            return this.resourceId.Name;
+            return this.Inner.Id;
         }
 
-        ///GENMHASH:7EF3D853483A47D9B9984C075548A5DB:740F056656F4A0D00BEAED4AA4FBF109
-        public int RequestedDatabaseDtuMax()
+        ///GENMHASH:AEE17FD09F624712647F5EBCEC141EA5:F31B0F3D0CD1A4C57DB28EB70C9E094A
+        public string State()
         {
-            return Inner.RequestedDatabaseDtuMax.GetValueOrDefault();
+            return this.Inner.State;
         }
 
-        ///GENMHASH:03526126BC38B1E6BD9561737558EC5D:BA5AF11F0F62FFF9B79F496BD38C3EB8
-        public string OperationId()
+        ///GENMHASH:E9EDBD2E8DC2C547D1386A58778AA6B9:9FE42D967416923E070F823D07063A47
+        public string ResourceGroupName()
         {
-            return Inner.OperationId;
+            return this.resourceId.ResourceGroupName;
         }
 
-        ///GENMHASH:8550B4F26F41D82222F735D9324AEB6D:42AE1A0453935D9BF88147F2F9C3EC20
-        public DateTime StartTime()
+        ///GENMHASH:90D0D26360B526D44FA6BF00B58572D2:6D05B62EB1B97540267134991BAE6456
+        public int RequestedDtuGuarantee()
         {
-            return Inner.StartTime.GetValueOrDefault();
+            return this.Inner.RequestedDtuGuarantee.GetValueOrDefault();
         }
 
-        ///GENMHASH:ACA2D5620579D8158A29586CA1FF4BC6:003E1843567E7760DFADC015F30E6AF4
-        public string Id()
+        ///GENMHASH:8F0EFF0AACCEFD64C58E15E16D3EC64E:337753FAB2FD3F556E28B3B41F66A995
+        public int RequestedDatabaseDtuMin()
         {
-            return this.resourceId.Id;
+            return this.Inner.RequestedDatabaseDtuMin.GetValueOrDefault();
         }
 
-        ///GENMHASH:3C1909F3137E91E93C57B90768BECD1A:561DFB636B157F511DFC1C40D1BDE39E
-        public DateTime EndTime()
+        ///GENMHASH:7EC64CE674517E507F9E7D72F93A7DF6:83C9FF1F82052A9E5127DEA47990F4A0
+        public string ErrorMessage()
         {
-            return Inner.EndTime.GetValueOrDefault();
+            //$ return this.Inner.ErrorMessage();
+
+            return null;
```

Comment:
this method contains commented java code. Is it fully implemented?

Reply:
fixed